core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +57,45 @@
 ])
 
 
+class RemoveDeprecatedExplorationModelFieldsOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Job that sets skill_tags, default_skin, skin_customization fields
+    in ExplorationModels to None in order to remove it from the datastore.
+    Job is necessary only for January 2021 release and can be removed after.
+    """
+
+    @classmethod
+    def enqueue(cls, job_id, additional_job_params=None):
+        super(RemoveDeprecatedExplorationModelFieldsOneOffJob, cls).enqueue(
+            job_id, shard_count=64)
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(exp_model):
+        removed_deprecated_field = False
+        for deprecated_field in [
+                'skill_tags', 'default_skin', 'skin_customizations']:
+            if deprecated_field in exp_model._properties:  # pylint: disable=protected-access
+                del exp_model._properties[deprecated_field]  # pylint: disable=protected-access
+                if deprecated_field in exp_model._values:  # pylint: disable=protected-access
+                    del exp_model._values[deprecated_field]  # pylint: disable=protected-access
+                removed_deprecated_field = True
```

Comment:
Should this be modified to, in order to fix the tests? We discused this at our meeting.
```suggestion
            if deprecated_field in exp_model._properties:  # pylint: disable=protected-access
                del exp_model._properties[deprecated_field]  # pylint: disable=protected-access
                removed_deprecated_field = True
            if deprecated_field in exp_model._values:  # pylint: disable=protected-access
                del exp_model._values[deprecated_field]  # pylint: disable=protected-access
            	removed_deprecated_field = True
```

Reply:
I modified it but the fields are still missing in `_properties`. The same errors are thrown. I have changed this and will be pushing today.

Reply:
Done, but still tests failing.